### PR TITLE
Maven only, with Kubernetes and DC/OS support

### PR DESCRIPTION
**Pull request by longshorej**
* Removes sbt build files, leaving this project as Maven only.
* Modifies `activity-stream-impl` to no longer depend on persistence/cluster.
* Attempts to behave similarly to LO4K8s, notably `reactive-lib` for service locator, and `akka-management` for cluster formation
* Includes hand-rolled configuration for Kubernetes and DC/OS, see the `deploy` folder.

This doesn't have steps to do the actual deployment. I think those are best covered in the guides (which should be updated).

### CODE REVIEW

File: activity-stream-impl/pom.xml [hunk 1]
```diff
@@ -33,14 +33,6 @@
...
             <artifactId>lagom-javadsl-server_${scala.binary.version}</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.lightbend.lagom</groupId>
-            <artifactId>lagom-javadsl-pubsub_${scala.binary.version}</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>com.lightbend.lagom</groupId>
-            <artifactId>lagom-javadsl-persistence-cassandra_${scala.binary.version}</artifactId>
-        </dependency>
```
Comment (ignasi35) [inline activity-stream-impl/pom.xml]: This is the _only_ change related to the PR title. 😂

**Comment by ignasi35** [comment]
Tested `mvn lagom:runAll`: LGTM except for a cosmetic issue. The UI looks odd:

screen_shot_2018-03-19_at_09_44_07

Could this be a by-product of an issue building the frontend assets (CSS, minified JS?,...)

**Comment by longshorej** [comment]
Thanks @ignasi35 - this is fixed, it was because foundation was upgraded which had style changes. I reverted to the old version for now (thus inline with the sbt version)

**Comment by longshorej** [comment]
This is ready for review, but we shouldn't merge it until a new `reactive-lib` is released.

/cc @mitkus

**Review by octonato — APPROVED** [review]

**Review by ignasi35 — APPROVED** [review]

**Comment by longshorej** [comment]
New `reactive-lib` is out, so this is ready for merge.

**Comment by octonato** [comment]
Let's do it!

**Pull request merged**